README.md

```diff
@@ -103,6 +103,14 @@ This runs [ESLint][] to discover problems within our codebase without executing
 
 This runs all the _lint_ commands at once.
 
+### `yarn olympia`
+
+A prerequisite to run this command is to [install addons-server locally](https://addons-server.readthedocs.io/en/latest/topics/install/index.html).
+
+This runs all development processes in a single column using [stmux][]. Open [http://olympia.dev:4000](http://olympia.dev:4000) to view the development site. **Press CTRL-a-? for help**. The application is configured to use a [local AMO API](https://addons-server.readthedocs.io/).
```

Comment:
These should all be `olympia.test` since that's currently the [preferred domain](https://addons-server.readthedocs.io/en/latest/topics/install/docker.html). The `.dev` one was problematic because it's an official domain.

Reply:
Oops, you're right. Typo..